Dynavity/Dynavity/view/MarkupView.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  MarkupView.swift
+//  Dynavity
+//
+//  Created by Sebastian on 14/3/21.
+//
+
+import SwiftUI
+
+struct MarkupView: View {
+    @ObservedObject var viewModel = MarkupViewModel()
+
+    var body: some View {
+        HStack {
+            TextEditor(text: $viewModel.markupTextBlock.rawText)
+            Divider()
+            WebView(rawHtml: viewModel.markupTextBlock.toHtml())
```

Comment:
Ideally, the `WebView` should only be scrollable if there is a need to and the rendered markup should be of the same size as the raw text. I think it might be easier to address this later on though; after we add the ability to add and resize canvas elements so that testing is easier.

Reply:
Yep, thanks for pointing them out. Will keep them in mind!